lib/scope.js

```diff
@@ -918,15 +918,19 @@ module.exports = {
       }
 
       let result = {
-        original: row,
+        // Unfortunately mutating frozen objects only fails silently.
```

Comment:
do we mind this? Trying to figure out why the comment is there, and what someone is supposed to do when they read it

Reply:
I imagine this is for me, or someone like me, ages from now who tries to mutate the object and nothing changes and such a person is confused about why. For a long time, despite being relatively competent with js, I didn't know this fact that seems pretty rarely used and obscure. I imagine tracing my way through the code and finding this clue to help. Certainly just reading `.freeze()` wouldn't have been enough even a short while ago. And what they're supposed to do is stop trying to mutate the object.

If/when we build a class for the table, I have plans to make things fail much less silently if devs try to mutate the object.

Does that all make sense, or am I still off base here? Our opinions may just be different about this. I'm somewhat attached to it, but if you feel strongly I can remove it.

Reply:
That makes sense. Personally, I would see `freeze()` and realize, "oh, I can't mutate it", but that makes sense, and couldn't still be confusing.

I think the "unfortunately" threw me off. Minor nit, but maybe "NOTE" would communicate the purpose better.

```suggestion
        // NOTE: mutating frozen objects will fail silently, so don't mutate this attribute
```